website/docs/podman/accessing-podman-from-another-wsl-instance.md

```diff
@@ -1,6 +1,6 @@
 ---
 sidebar_position: 60
-title: Accessing Podman from another WSL distribution
+title: Accessing Podman from another WSL distribution (Windows)
 description: On Windows, access your Podman Desktop containers from another Windows Subsystem for Linux (WSL) distribution.
 ---
```

Comment:
Can we also add keywords and tags here?

Reply:
honestly the tags or keywords don't do much so I avoided it. i'll update in a different pr!